JamfSync/Model/JsonObjects.swift

```diff
@@ -239,3 +239,41 @@ struct JsonInitiateUpload: Decodable {
 struct JsonJamfProVersion: Decodable {
     let version: String
 }
+
+// Added - BIG-RAT
+struct Headers {
+    static var canonical = ""
+    static var signed    = ""
+}
+
+class UploadTime {
+    var start: Int
+    var end: Int
+    
+    init(start: Int, end: Int) {
+        self.start = start
+        self.end = end
+    }
+    
+    func total() -> String {
```

Comment:
this function feels really inefficient to me. You should probably do something with `DateComponentsFormatter` like described [here](https://cocoacasts.com/cocoa-fundamentals-formatting-a-time-interval-in-swift)